new-site/src/components/layout.js

```diff
@@ -86,6 +86,18 @@ const resolveNavigationLinkByPathAndLevel = (parentPath, level) => (page) => {
   return pathParts.length === level && pathParts.slice(0, -1).every((pathPart) => parentPath.includes(pathPart));
 };
 const sortByPageTitle = (pageA, pageB) => pageA.title.localeCompare(pageB.title);
+const isMatchingParentLink = (link, slug) => {
+  const slugParts = resolvePathPartsFrom(slug);
+  const linkParts = resolvePathPartsFrom(link);
+  const slugPartsWithoutLast = slugParts.slice(0, -1);
+
+  return (
+    slugPartsWithoutLast.length >= 2 &&
```

Comment:
We might be able to simplify this like so:

```
slugPartsWithoutLast.length === linkParts.length &&
slugPartsWithoutLast.length >= 2
```

Check equality first, after that we only need to check the length of one (linkParts or slugPartsWithoutLast)

Reply:
Good catch 👍 
- [ ] fix condition